i18n/locales/source/hourofcode/en.yml

```diff
@@ -11356,7 +11356,7 @@
   about_page_section_more_description_1: "In 2013, Code.org was launched by twin brothers [Hadi](%{hadi_url}) and [Ali Partovi](%{partovi_url}) with a [video](%{video_url}) promoting computer science. This video became #1 on YouTube for a day, and 15,000 schools reached out to us for help. Since then, we've expanded from a bootstrapped staff of volunteers to build a full organization supporting a worldwide movement. We believe that a quality computer science education should be available to every child, not just a lucky few."
   about_page_section_more_description_2: "To support our goal, we do work across the education spectrum: designing our own courses or partnering with others, training teachers, partnering with large school districts, helping change government policies, expanding internationally via partnerships, and marketing to break stereotypes."
   about_page_section_more_description_3: "Our work builds upon [decades of effort, by countless organizations and individuals](%{efforts_url}) who have helped establish, fund, and spread computer science education. We're thankful to benefit from the tireless help of the broader computer science education community, and [we thank all the partners and individuals who have supported our impact over the years](%{partners_url}))."
-  about_page_annual_report_2022: "[Code.org 2022 Annual Report](%{report_url_2022}) (Past reports: [2021](%{report_url_2021}), [2020](%{report_url_2020}), [2019](%{report_url_2019}), [2018](%{report_url_2018}), [2017](%{report_url_2017}), [2016](%{report_url_2016}), [2015](%{report_url_2015}), [2014](%{report_url_2014}))"
+  about_page_annual_report_2023: "[Code.org 2023 Annual Report](%{report_url_2023}) (Past reports: [2022](%{report_url_2022}), [2021](%{report_url_2021}), [2020](%{report_url_2020}), [2019](%{report_url_2019}), [2018](%{report_url_2018}), [2017](%{report_url_2017}), [2016](%{report_url_2016}), [2015](%{report_url_2015}), [2014](%{report_url_2014}))"
```

Comment:
This actually needs to be edited in [this file](https://github.com/code-dot-org/code-dot-org/blob/6d98463116fcb0b05177e948fb521ad5a90eb00e/pegasus/sites.v3/hourofcode.com/i18n/en.yml#L11359) instead.